Gruntfile.js

```diff
@@ -1,21 +1,29 @@
 module.exports = function(grunt) {
-
+  var webpack = require('webpack');
   grunt.initConfig({
     pkg: grunt.file.readJSON('package.json'),
-    uglify: {
-      options: {
-        banner: '/*! <%= pkg.name %> <%= pkg.version %> <%= grunt.template.today("yyyy-mm-dd") %> */\n',
-        report: 'min'
-      },
-      dist: {
-        files: {
-          '<%= pkg.name %>.min.js': ['<%= pkg.name %>.js']
-        }
+
+    webpack: {
+      backboneEvents: {
+        entry: "./backbone-events-standalone.js",
```

Comment:
The file is now mixing double and single quotes. Think it's better to stick with single quotes throughout the project.